kafka_utils/util/client.py

```diff
@@ -59,3 +67,86 @@ def send_offset_commit_request_kafka(
 
         return [resp if not callback else callback(resp) for resp in resps
                 if not fail_on_error or not self._raise_on_response_error(resp)]
+
+    def send_consumer_metadata_request(self, payloads=[], fail_on_error=True,
+                                       callback=None):
```

Comment:
possibly add a docstring since this is overridden, so it might be good to say what does this do differently. Also we should probably add some tests for this, since this is changing some functionality.